File: kino24-news-read/src/main/java/com/kino24/kino24newsread/core/domain/entity/Like.java
package com.kino24.kino24newsread.core.domain.entity;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;
import java.time.ZonedDateTime;
import java.util.UUID;

@Data
@Entity
@Table(name = "likes")
public class Like {
    @Id
    private UUID id;

    private UUID userId;

    @ManyToOne
    @JoinColumn(name = "article_id")
    private Article article;

    @ManyToOne
    @JoinColumn(name = "comment_id")
    private Comment comment;

    private ZonedDateTime timestamp;
}

File: kino24-like/kino24-like.Core/Entities/Like/Like.cs
using System.ComponentModel.DataAnnotations;

namespace kino24_like.Core.Entities
{
    public class Like 
    {
        public Guid ID { get; set; }

        public string UserId { get; set; }

        public LikeReaction Reaction { get; set; }

        public string? ArticleId { get; set; }

        public string? CommentId { get; set; }
    }
}
